addons/mail/static/src/models/message/message.js

```diff
@@ -574,6 +587,12 @@ function factory(dependencies) {
         author: many2one('mail.partner', {
             inverse: 'messagesAsAuthor',
         }),
+        sender: many2one('mail.partner', {
+            inverse: 'messagesAsSender',
+        }),
+        author_has_user: attr({
+            default: true,
+        }),
```

Comment:
Missing docs on these fields (briefly explain what they are conceptually, could give a hint on the difference between an author and a sender, might be appropriate to document `author` too)